### PR TITLE
add landing page for e-commerce site

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

**Comment by ganesh-kanchi** [comment]
> _ No description provided. _

here is the deployed link
https://jacket-ex.netlify.app/

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    opacity: 0.5;
+    position: absolute;
+    z-index: -1;
+    object-fit: cover;
+    
+}
+
+.banner-background{
+}
+
```
Comment (mariyasada) [inline styles.css]: remove this unnecessary class

**Review by mariyasada — APPROVED** [review]
look at the comment

File: styles.css (hunk 1 above)
Comment (ganesh-kanchi) [reply styles.css]: ok will do

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+.banner-content{
+    display: flex;
+    justify-content: center;
```
Comment (paarthbajaj) [inline styles.css:46]: You can make a utility class for flex and use it here or whenever you want.

**Review by paarthbajaj — APPROVED** [review]
Please look at the color contrast of the footer. The visibility is not so good. Also, try making utility classes if you have not started making them.

File: styles.css (hunk 2 above)
Comment (ganesh-kanchi) [reply styles.css:46]: ok will use that.

**Pull request merged**